cross/arm/toolchain.cmake

```diff
@@ -4,23 +4,15 @@ set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_VERSION 1)
 set(CMAKE_SYSTEM_PROCESSOR armv7l)
 
-## Specify the toolchain
 set(TOOLCHAIN "arm-linux-gnueabihf")
-set(TOOLCHAIN_PREFIX ${TOOLCHAIN}-)
-#set(CMAKE_C_COMPILER ${TOOLCHAIN_PREFIX}gcc)
-#set(CMAKE_CXX_COMPILER ${TOOLCHAIN_PREFIX}g++)
-#set(CMAKE_ASM_COMPILER ${TOOLCHAIN_PREFIX}as)
-#set(CMAKE_AR ${TOOLCHAIN_PREFIX}ar)
-#set(CMAKE_OBJCOPY ${TOOLCHAIN_PREFIX}objcopy)
-#set(CMAKE_OBJDUMP ${TOOLCHAIN_PREFIX}objdump)
 
 add_compile_options(-target armv7-linux-gnueabihf)
 add_compile_options(-mthumb)
 add_compile_options(-mfpu=vfpv3)
 add_compile_options(--sysroot=${CROSS_ROOTFS})
 
 set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -target ${TOOLCHAIN}")
-set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -B ${CROSS_ROOTFS}/usr/lib/gcc/${TOOLCHAIN}")
+set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -B${CROSS_ROOTFS}/usr/lib/gcc/${TOOLCHAIN}")
```

Comment:
We do the same on Android (to keep the files consistent), hence `${TOOLCHAIN}` must be an absolute path, so we can't prefix `${TOOLCHAIN}` like we could do with `${TOOLCHAIN_PREFIX}`